kong/cmd/config.lua

```diff
@@ -157,8 +155,10 @@ Usage: kong config COMMAND [OPTIONS]
 Use declarative configuration files with Kong.
 
 The available commands are:
-  init                                Generate an example config file to
-                                      get you started.
+  init [<file>]                       Generate an example config file to
+                                      get you started. If a filename
+                                      is not given, ]] .. INIT_FILE .. [[ is used
+                                      by default.
```

Comment:
ditto: hint that `INIT_FILE` will be written in the current directory, while `<file>` can be a path.

Reply:
Can do, but feeling uninspired since the message as is seems obvious enough to my biased eyes — how do you suggest wording that?

(Note also that other subcommands below also accept `<file>` with full pathnames but none of them mention that; I always assumed this as a given.)